docs/logs/quick-start.md

```diff
@@ -0,0 +1,102 @@
+title: Sematext Logs Quick Start
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
+
+After you get logged into Sematext Cloud at <https://apps.sematext.com> (or <https://apps.eu.sematext.com> if using Sematext Cloud Europe), the first step is to create a Logs App. An App is an independent namespace for your data.
```

Comment:
Does it make sense to link something to "What's an App?" ?  Or maybe at the end of the paragraph?

Reply:
Added link